extensions/fix-your-indent/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "fix-your-indent",
+  "title": "Fix Your Indent",
+  "description": "Removes unnecessary leading whitespaces from text when you want to paste code into your web browser",
+  "icon": "command-icon.png",
+  "author": "csharp",
+  "categories": [
+    "Developer Tools",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Remove Unnecessary Leading Whitespaces",
+      "description": "Removes unnecessary leading whitespaces from text when you want to paste code into your web browser",
+      "mode": "no-view"
+    }
+  ],
+  "tools": [],
+  "dependencies": {
+    "@raycast/api": "^1.76.0"
+  },
+  "devDependencies": {
+    "@raycast/eslint-config": "^1.0.6",
+    "@types/node": "20.8.10",
+    "@types/react": "18.2.27",
+    "eslint": "^8.51.0",
+    "prettier": "^3.0.3",
+    "typescript": "^5.2.2"
+  },
+  "scripts": {
+    "build": "ray build -e dist",
+    "dev": "ray develop",
+    "fix-lint": "ray lint --fix",
+    "lint": "ray lint",
+    "prepublishOnly": "echo \"\\n\\nIt seems like you are trying to publish the Raycast extension to npm.\\n\\nIf you did intend to publish it to npm, remove the \\`prepublishOnly\\` script and rerun \\`npm publish\\` again.\\nIf you wanted to publish it to the Raycast Store instead, use \\`npm run publish\\` instead.\\n\\n\" && exit 1",
```

Comment:
```suggestion
```
I don't think this is needed

Reply:
This was just there by default when I ran the `Create Extension` command. I'll delete it!